tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -149,6 +166,50 @@ func NewStresser(s string, sc *stressConfig, m *member) Stresser {
 			keysPerLease: sc.keysPerLease,
 			rateLimiter:  sc.rateLimiter,
 		}
+	case "election-runner":
+		return &runnerStresser{
+			cmdStr: sc.etcdRunnerPath,
+			args: []string{
+				"election",
+				"--rounds=10000000", // large enough to run a long time.
```

Comment:
I feel like etcd-runner should treat --rounds=0 as running forever instead of needing this hack